tests/python_tests/test_vlm_pipeline.py

```diff
@@ -27,13 +27,19 @@ def get_ov_model(model_id):
         trust_remote_code=True,
         **align_with_optimum_cli,
     ))
-    processor.tokenizer.save_pretrained(model_dir)
-    ov_tokenizer, ov_detokenizer = openvino_tokenizers.convert_tokenizer(processor.tokenizer, with_detokenizer=True)
+    # For tiny-random-internvl2 processor is actually tokenizer
+    if isinstance(processor, transformers.Qwen2TokenizerFast):
+        tokenizer = processor
+        processor = transformers.AutoImageProcessor.from_pretrained(model_id, trust_remote_code=True)
+    else:
+        tokenizer = processor.tokenizer
```

Comment:
@eaidova For `tiny-random-internvl2` model we have the issue: `AttributeError: Qwen2TokenizerFast has no attribute tokenizer`.
As I can see in transformers, no proper processor is found for this model in mappings, so it falls back to `AutoTokenizer`. As a result no image processor component created and `preprocessor_config.json` is not saved. So here I am explicitly using `AutoImageProcessor` as processor to save all components correctly.
Any objections or thoughts on the fix?